platform/eureka/client.go

```diff
@@ -0,0 +1,97 @@
+package eureka
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"time"
+)
+
+type application struct {
+	Name      string      `json:"name"`
+	Instances []*instance `json:"instance"`
+}
+
+type instance struct {
+	Hostname   string   `json:"hostName"`
+	App        string   `json:"app"`
+	IPAddress  string   `json:"ipAddr"`
+	Status     string   `json:"status"`
```

Comment:
Yes. We filter out the non-"UP" instances in the conversion logic (in `conversion.go`).